keps/sig-windows/2578-windows-conformance/kep.yaml

```diff
@@ -0,0 +1,34 @@
+title: Windows Conformance
+kep-number: 2578
+authors:
+  - "@vladimirvivien"
+  - "@jayunit100"
+  - "@perithompson"
+  - "@johnSchnake"
+  - "@claudiubelu"
+  - "@knabben"
+  - "@rosskirkpat"
+  - "@hxietkg"
+owning-sig: sig-windows
+participating-sigs:
+  - sig-testing
```

Comment:
Was this KEP brought up in SIG Testing at all? When/Where?

Reply:
@knabben @jayunit100 @claudiubelu

Reply:
@BenTheElder I did bring it up a lonnnng time ago - the feedback was that
there shouldnt be a "Conformance" Specification - hence we defined this as an operational readiness specification, and are updating alot of the sig-testing/ owned tests to support it in a way that is orthogonal. 

We can remove the participating-sigs bullet if you want though?